packages/table-module/src/module/with-selection.ts

```
@@ -52,76 +66,135 @@ export function withSelection<T extends Editor>(editor: T) {
+      if (!filled || filled.length === 0) {
+        TableCursor.unselect(editor)
+        return apply(op)
+      }
 
-        if (Path.equals(fromPath, path)) {
-          from.x = x
-          from.y = y
+      // 添加调试信息
+      console.log('=== 表格选择调试 ===')
+      console.log('fromPath:', fromPath.join(','))
+      console.log('toPath:', toPath.join(','))
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)